docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -50,11 +50,16 @@ You can extend your rules by connecting them to actions that use the following s
 [role="screenshot"]
 image::images/alert-action-types.png[Action types]
 
-When configuring an action type, you can define precisely when the alert is triggered by selecting a specific
+After you select a connector, you must set the action frequency. You can choose to create a summary of alerts on each check interval or on a custom interval. For example, send email notifications that summarize the new, ongoing, and recovered alerts each hour:
+
+[role="screenshot"]
+image::images/action-alert-summary.png[Action types]
+
+Alternatively, you can set the action frequency such that you choose how often the action runs (for example, at each check interval, only when the alert status changes, or at a custom action interval). In this case, you define precisely when the alert is triggered by selecting a specific
 threshold condition: `Alert`, `Warning`, or `Recovered` (a value that was once above a threshold has now dropped below it).
 
 [role="screenshot"]
-image::images/run-when-selection.png[Configure when an alert is triggered]
```

Comment:
Can we delete the `run-when-selection.png` screenshot since it's no longer being used?

Reply:
I've renamed it, since that run-when-selection.png was still being used by the metric threshold rule page.

Reply:
Aha! Thanks!